commit 008b46a1d3e3a1a9459590be86a4c0014e333976
Author: Aki Tuomi <aki.tuomi@dovecot.fi>
Date:   Wed Dec 14 10:32:44 2016 +0200

    sha3: Fix typo in ifdef
    
    The code was supposed to be used with big endian
    machines.
    
    Reported by than@redhat.com

diff --git a/src/lib/sha3.c b/src/lib/sha3.c
index 8cf61fa..67c4476 100644
--- a/src/lib/sha3.c
+++ b/src/lib/sha3.c
@@ -232,7 +232,7 @@ sha3_finalize(struct sha3_ctx *ctx)
 			SHA3_CONST(0x8000000000000000UL);
 	keccakf(ctx->s);
 
-#ifndef WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	{
 		unsigned i;
 		for(i = 0; i < SHA3_KECCAK_SPONGE_WORDS; i++) {
